lang/de/settings.php

```diff
@@ -5,7 +5,7 @@
  *
  * @author Markus Hoffrogge <mhoffrogge@gmail.com>
  */
-
+$lang['pingPushHost'] = 'Ping the push host, to make sure it is available before pushing. empty to disable';
```

Comment:
Please change to German: `Ping den Push Server zur Prüfung der Verfügbarkeit vor einem push. Inaktiv, wenn leer.`